test/test_grid_file.py

```diff
@@ -380,29 +411,30 @@ def test_multi_chunk_file(self):
         g = GridOut(self.db.fs, f._id)
         self.assertEqual(random_string, g.read())
 
-    def test_small_chunks(self):
-        self.files = 0
-        self.chunks = 0
-
-        def helper(data):
-            f = GridIn(self.db.fs, chunkSize=1)
-            f.write(data)
-            f.close()
-
-            self.files += 1
-            self.chunks += len(data)
-
-            self.assertEqual(self.files, self.db.fs.files.count_documents({}))
-            self.assertEqual(self.chunks, self.db.fs.chunks.count_documents({}))
-
-            g = GridOut(self.db.fs, f._id)
-            self.assertEqual(data, g.read())
-
-            g = GridOut(self.db.fs, f._id)
-            self.assertEqual(data, g.read(10) + g.read(10))
-            return True
-
-        qcheck.check_unittest(self, helper, qcheck.gen_string(qcheck.gen_range(0, 20)))
+    # # TODO: convert qcheck to async
```

Comment:
New ticket?

Reply:
https://jira.mongodb.org/browse/PYTHON-4708, put it in the comment.